### PR TITLE
997 issue page customisation

### LINKED ISSUE #997: Further customisation options for the issues page
**Is your feature request related to a problem? Please describe.**
Users would like to customise the issue page blocks:
 - Remove the issue number
 - Render the year of the issue(conference) in place of the volume number

### PR DESCRIPTION
**Pull request by ajrbyers**
Closes #997

### CODE REVIEW

File: src/journal/logic.py [hunk 1]
```diff
@@ -483,3 +483,35 @@ def potential_issue_editors(journal, current_editors):
...
+    if sort_type == 'date_sort_desc':
+        order = '-date'
+    else:
+        order = 'date'
+
+    ordered_issues = issue_list.order_by(order)
+
+    pks = [issue.pk for issue in ordered_issues]
+
+    for issue in issue_list:
```
Comment (mauromsl) [inline src/journal/logic.py]: If you iterate `enumerate(ordered_issues)` instead you avoid the intermediate list & lookup

File: src/themes/OLH/templates/journal/issues.html [hunk 2]
```diff
@@ -20,10 +20,9 @@ <h3>{% trans 'Issues' %}</h3>
                         <a href="{% url 'journal_issue' issue.id %}" class="box-link"></a>
                         <img src="{% if issue.cover_image %}{{ issue.cover_image.url }}{% elif journal.default_cover_image %}{{ journal.default_cover_image.url }}{% else %}{% static "common/img/sample/issue_cover.png" %}{% endif %}"
                              alt="{{ issue.title }}">
-                        <span class="volume-number"><span>{% trans "Volume" %}</span> {{ issue.volume }}</span>
+
```
[the remark this answers is not in the document]
Comment (ajrbyers) [reply src/themes/OLH/templates/journal/issues.html]: Yeah, it actually sits on top of text as well so I thought it best to remove it.

File: src/journal/models.py [hunk 3]
```diff
@@ -397,7 +403,7 @@ class Issue(models.Model):
     issue = models.IntegerField(default=1)
     issue_title = models.CharField(blank=True, max_length=300)
     date = models.DateTimeField(default=timezone.now)
-    order = models.IntegerField(default=1)
+    order = models.IntegerField(default=0)
```
[the remark this answers is not in the document]
Comment (ajrbyers) [reply src/journal/models.py]: Indeed!

File: src/journal/logic.py (hunk 1 above)
Comment (ajrbyers) [reply src/journal/logic.py]: Noice. :+1:

**Comment by ajrbyers** [comment]
Added commit to address comments.

File: src/journal/logic.py [hunk 4]
```diff
@@ -483,3 +483,35 @@ def potential_issue_editors(journal, current_editors):
...
+        return
+
+    if sort_type == 'date_sort_desc':
+        order = '-date'
+    else:
+        order = 'date'
+
+    ordered_issues = issue_list.order_by(order)
+
+    pks = [issue.pk for issue in ordered_issues]
```
Comment (mauromsl) [inline src/journal/logic.py]: You forgot to delete this :P

Comment (ajrbyers) [reply src/journal/logic.py]: Its for decoration!

**Pull request merged**